server/bin/deploy

```diff
@@ -178,10 +178,11 @@ init
 
 DBUSER="candlepin"
 
-while getopts ":fgtolmav" opt; do
+while getopts ":fghtolmav" opt; do
```

Comment:
Commenting here due to a shortcoming in Github

Don't forget to update the help output with the new option details.